client/src/components/containers/CodeReviewMetricsContainers/CodeReviewMetricsTable/CodeReviewCSVDownloader/CodeReviewCSVDownloaderUtils.ts

```diff
@@ -0,0 +1,31 @@
+import { NOT_AVAILABLE, SENTENCE_JOINER } from "src/constants/constants";
+import {
+  IFetchedPullRequestComments,
+  IFetchedPullRequestVotesTimeline,
+  IFetchedPullRequestReviewerComments,
+} from "src/services/api/api";
+import { getFormattedDateWithTime } from "src/utils/dateUtil";
+
+export class CodeReviewCSVDownloaderUtils {
+  static getGeneralComments(comments: IFetchedPullRequestComments) {
+    return comments.totalComments - comments.numberOfMajorComments - comments.numberOfNitComments;
+  }
+
+  static getFormattedVotesTimeline(votesTimeline: IFetchedPullRequestVotesTimeline[]) {
+    if (!votesTimeline) {
+      return NOT_AVAILABLE;
+    }
+
+    return votesTimeline
+      .map((vote) => `${vote.author} (${vote.vote} - ${getFormattedDateWithTime(vote.timeOfVote)})`)
+      .join(SENTENCE_JOINER);
```

Comment:
If this variable is used only here . Can we declare it as private variable inside this ?

Reply:
It has been used in component too, so I made declared it in common constanmts.

Reply:
This is moved to component constant file.